python/private/pypi/generate_whl_library_build_bazel.bzl

```diff
@@ -14,406 +14,64 @@
-        additional_content.append(config_settings_content)
-
-    lib_dependencies = _render_list_and_select(
-        deps = dependencies,
-        deps_by_platform = dependencies_by_platform,
-        tmpl = dep_template.format(name = "{}", target = PY_LIBRARY_PUBLIC_LABEL),
-    )
-
-    whl_file_deps = _render_list_and_select(
-        deps = dependencies,
-        deps_by_platform = dependencies_by_platform,
-        tmpl = dep_template.format(name = "{}", target = WHEEL_FILE_PUBLIC_LABEL),
-    )
-
-    # If this library is a member of a group, its public label aliases need to
-    # point to the group implementation rule not the implementation rules. We
-    # also need to mark the implementation rules as visible to the group
-    # implementation.
-    if group_name and "//:" in dep_template:
-        # This is the legacy behaviour where the group library is outside the hub repo
-        label_tmpl = dep_template.format(
-            name = "_groups",
-            target = normalize_name(group_name) + "_{}",
-        )
-        impl_vis = [dep_template.format(
-            name = "_groups",
-            target = "__pkg__",
-        )]
-        additional_content.extend([
-            "",
-            render.alias(
-                name = PY_LIBRARY_PUBLIC_LABEL,
-                actual = repr(label_tmpl.format(PY_LIBRARY_PUBLIC_LABEL)),
-            ),
-            "",
-            render.alias(
-                name = WHEEL_FILE_PUBLIC_LABEL,
-                actual = repr(label_tmpl.format(WHEEL_FILE_PUBLIC_LABEL)),
-            ),
-        ])
-        py_library_label = PY_LIBRARY_IMPL_LABEL
-        whl_file_label = WHEEL_FILE_IMPL_LABEL
-
-    elif group_name:
-        py_library_label = PY_LIBRARY_PUBLIC_LABEL
-        whl_file_label = WHEEL_FILE_PUBLIC_LABEL
-        impl_vis = [dep_template.format(name = "", target = "__subpackages__")]
-
-    else:
-        py_library_label = PY_LIBRARY_PUBLIC_LABEL
-        whl_file_label = WHEEL_FILE_PUBLIC_LABEL
-        impl_vis = ["//visibility:public"]
-
     contents = "\n".join(
         [
-            _BUILD_TEMPLATE.format(
-                loads = "\n".join(sorted(loads)),
-                py_library_label = py_library_label,
-                dependencies = render.indent(lib_dependencies, " " * 4).lstrip(),
-                whl_file_deps = render.indent(whl_file_deps, " " * 4).lstrip(),
-                data_exclude = repr(_data_exclude),
-                whl_name = whl_name,
-                whl_file_label = whl_file_label,
-                tags = repr(tags),
-                data_label = DATA_LABEL,
-                dist_info_label = DIST_INFO_LABEL,
-                entry_point_prefix = WHEEL_ENTRY_POINT_PREFIX,
-                srcs_exclude = repr(srcs_exclude),
-                data = repr(data),
-                impl_vis = repr(impl_vis),
-            ),
+            """load("@rules_python//python/private/pypi:whl_library_targets.bzl", "whl_library_targets")""",
```

Comment:
Since we've basically gutted this file, how about moving this to be a template string, similar to how we've switched to doing elsewhere?

```
_BUILD_TEMPLATE = """
load("@rules_python//python/private/pypi:whl_library_targets.bzl", "whl_library_targets")

package(default_visibility = ["//visibility:public"])

whl_library_targets(
{kwargs}
)
"""

_BUILD_TEMPLATE.format(render.kwargs(kwargs))

```

(render.kwargs just popped into my head, a `"\n".join([...])` also SGTM)